src/routes/hooks/hooks-cache.controller.spec.ts

```diff
@@ -36,6 +36,7 @@ import { PostgresDatabaseModule } from '@/datasources/db/v1/postgres-database.mo
 import { TestPostgresDatabaseModule } from '@/datasources/db/__tests__/test.postgres-database.module';
 
 // TODO: Migrate to E2E tests as TransactionEventType events are already being received via queue.
+// Add *_DELEGATE event tests here if we unskip this
```

Comment:
We should include tests for `*_DELEGATE` events in `event-hooks-queue.e2e-spec`, right?

Reply:
Added in 8f0f09ba2e3264f8db2742ecab64f0ac85bcf967.

Reply:
Awesome, thanks!